test/FixCode/fixits-apply.swift

```diff
@@ -1,6 +1,8 @@
 // RUN: not %swift -parse -target %target-triple %s -emit-fixits-path %t.remap -I %S/Inputs
 // RUN: c-arcmt-test %t.remap | arcmt-test -verify-transformed-files %s.result
 
+// REQUIRES: objc_interop
```

Comment:
Can you split the file into objc and non-objc parts, so that we don't lose coverage on Linux?